src/main/frontend/db/react.cljs

```diff
@@ -4,15 +4,62 @@
   It'll be great if we can find an automatically resolving and performant
   solution.
   "
-  (:require [clojure.string :as string]
-            [datascript.core :as d]
-            [frontend.config :as config]
+  (:require [datascript.core :as d]
             [frontend.date :as date]
             [frontend.db.conn :as conn]
             [frontend.db.utils :as db-utils]
             [frontend.state :as state]
             [frontend.util :as util :refer [react]]
-            [frontend.util.marker :as marker]))
+            [frontend.db-schema :as db-schema]
+            [cljs.spec.alpha :as s]))
+
+;;; keywords specs for reactive query, used by `react/q` calls
```

Comment:
Glad to see more validation. Are we planning on using spec for the longer term? If so, I would encourage us to move specs to their own namespaces as it allows for their reuse in other contexts. I can create a card for this if we are sticking with spec

Reply:
Not sure whether we're sticking with spec. I also like https://github.com/metosin/malli for its data-driven schemas. The only sure thing is that we need to add more spec/schema 😄